modules/cluster-limitations-local-zone.adoc

```diff
@@ -17,3 +17,14 @@ The following list details limitations when deploying a cluster in AWS Local Zon
 - Network resources such as Network Load Balancer (NLB), Classic Load Balancer, and Network Address Translation (NAT) Gateways are not supported in AWS Local Zones.
 - For an {product-title} cluster on AWS, the AWS Elastic Block Storage (EBS) `gp3` type volume is the default for node volumes and the default for the storage class. This volume type is not globally available on Local Zone locations. By default, the nodes running in Local Zones are deployed with the `gp2` EBS volume. The `gp2-csi` `StorageClass` must be set when creating workloads on Local Zone nodes.
 ====
+
+If you want the installation program to automatically create Local Zone subnets for your {product-title} cluster, specific configuration limitations apply with this method.
+
+[IMPORTANT]
+====
+The following list details configuration limitations that apply when you set the installation program to automatically create subnets for your {product-title} cluster:
+
+- A network plugin, which is specific to an overlay network, automatically adjusts the MTU value for an overlay network. With the installation program automatically creating subnets on your behalf, you cannot adjust the MTU value for the overlay network.
+- A network plugin uses the Network Address Translation (NAT) Gateway from the route table of the parent zone to route egress traffic for private subnets in an AWS Local Zone. If a parent zone does not exist in your VPC, the network plugin uses the NAT gateway listed in the first available private subnet listed in an AWS region.
+- An Ingress Controller must shard Classless Inter-Domain Routing (CIDR) Machine Network blocks for a VPC to smaller blocks if you listed a large number of zone locations in the installation configuration file. Sharding ensures that the installation program can create all the necessary private and public subnets in each Local Zone location.
```

Comment:
Could you please elaborate on this paragraph?

Reply:
This content only exists in this PR and is not published to the customer docs. I cannot determine where this parapaph originated from, but maybe I pulled some information from https://docs.openshift.com/container-platform/4.13/networking/ingress-sharding.html and CIDR references in [install_local-zones.md](https://github.com/openshift/installer/blob/bee55594fe2b56ab0ed7925c5116dd3db45d0cfd/docs/user/aws/install_local-zones.md#cluster-installation-in-aws-extending-nodes-to-local-zones-)

Remove this paragraph?